src/ValidationModels/penaltyValidation.js

```diff
@@ -21,14 +23,14 @@ const trailerSchema = {
 
 const valueSchema = {
 	penaltyType: Joi.string().regex(/^(IM|CDN|FPN)$/).required(),
-	paymentStatus: Joi.string(), // .regex(/^(UNPAID|PAID)$/).required(),
-	paymentAuthCode: Joi.string(),
+	paymentStatus: Joi.string().valid(['PAID', 'UNPAID']),
```

Comment:
I see `.required` is gone here, is that intentional?

Reply:
Yes, `.required` was commented out and the app uploads documents without a payment status.